connector_prestashop/tests/test_export_stock_qty_job.py

```diff
@@ -59,12 +59,12 @@ def test_job_export_qty(self):
 
             self.assertTrue(
                 set({'depends_on_stock': '0',
-                     'id': '1',
-                     'id_product': '1',
+                     'id': '2',
+                     'id_product': '2',
                      'id_product_attribute': '0',
                      'id_shop': '1',
                      'id_shop_group': '0',
-                     'out_of_stock': '2',
+                     'out_of_stock': '0',
```

Comment:
```
FAIL: test_job_export_qty (odoo.addons.connector_prestashop.tests.test_export_stock_qty_job.TestExportStockQuantity)
` Export a qty on PrestaShop
Traceback (most recent call last):
`   File "connector_prestashop/tests/common.py", line 71, in _decorated
`     result = func(self, *args, **kwargs)
`   File "connector_prestashop/tests/test_export_stock_qty_job.py", line 69, in test_job_export_qty
`     .issubset(set(body['prestashop']['stock_available'].items())))
` AssertionError: False is not true
```

_If stock shortage_ default value is False instead of _Default prestashop_